src/signals/IncidentMap/components/FilterPanel/styled.ts

```diff
@@ -50,3 +47,66 @@ export const StyledButton = styled(Button)`
     }
   }
 `
+
+export const SubSection = styled.div<{ visible: boolean; lines: number }>`
+  padding-left: ${themeSpacing(4)};
+  max-height: 0;
+  transition: max-height 0.15s ease-out;
+  overflow: hidden;
+
+  ${({ visible, lines }) =>
+    visible &&
+    /*lines time 5 because line wrapped labels, setting needs to be above real height (to avoid clipping) but way too
+  high gives wonky expand/collapse effect. See https://css-tricks.com/using-css-transitions-auto-dimensions/*/
+    css`
+      max-height: ${5 * lines}em;
+      transition: max-height 0.25s ease-in;
+    `}
+`
+
+export const SectionWrapper = styled.div``
+
+export const WrapperFilterCategoryWithIcon = styled.div`
+  > label {
+    width: calc(100% - 20px);
+  }
+
+  display: flex;
+  align-items: center;
+`
+
+export const InvisibleButton = styled.button<{ toggle: boolean }>`
+  text-decoration: none;
+  background-color: unset;
+  color: inherit;
+  border: none;
+  padding: 0px;
+
+  > * {
+    transition: transform 0.25s;
+
+    ${({ toggle }) =>
+      toggle &&
+      css`
+        transform: rotate(180deg);
+      `}
+  }
+`
+
+interface LegendTextProps {
+  readonly noResults?: boolean
+}
+
+export const CategoryItemText = styled.span<LegendTextProps>`
+  display: flex;
+  align-items: center;
+  color: ${(props) =>
+    props.noResults
+      ? themeColor('tint', 'level4')
+      : themeColor('tint', 'level7')};
+`
+
```

Comment:
This is interesting but not used:
- the noResult prop is there to let the styling know if the filter has any incidents. If not, you can pass this prop and render the text in grey. Maybe we can use this too? (see screenshot)
- "LegendTextProps" is a name from somewhere else. 


<img width="617" alt="Screenshot 2022-11-01 at 11 07 22" src="https://user-images.githubusercontent.com/46756331/199209833-47fcb926-747c-4ffa-a938-919e03ea7826.png">